### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by adrian-lorenzo**
Changes:

- **Added documentation** (markdown files) to `source/includes`

### CODE REVIEW

File: source/includes/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Booster Documentation
+
+1. [Getting Started](_01-getting-started.md): Setup your environment to start creating Booster applications.
+2. [Architecture](_02architecture.md): Understand the high-level Booster Cloud Framework architecture.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
2. [Architecture](_02-architecture.md): Understand the high-level Booster Cloud Framework architecture.
```

File: source/includes/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+3. [Commands](_03commands.md): Design the Write API of your application.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
3. [Commands](_03-commands.md): Design the Write API of your application.
```

File: source/includes/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+4. [Events](_04events.md): Understand the source of truth of your application.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
4. [Events](_04-events.md): Understand the source of truth of your application.
```

File: source/includes/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+5. [Entities](_05entities.md): Design the domain model of your application.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
5. [Entities](_05-entities.md): Design the domain model of your application.
```

File: source/includes/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+6. [Read Models](_06read-models.md): Design the Read API of your application.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
6. [Read Models](_06-read-models.md): Design the Read API of your application.
```

File: source/includes/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+7. [Authentication and Authorization](_07auth.md): Limiting access.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
7. [Authentication and Authorization](_07-auth.md): Limiting access.
```

File: source/includes/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+8. [Deployment](_08deployment.md): Deploy your application!
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
8. [Deployment](_08-deployment.md): Deploy your application!
```

File: source/includes/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+9. [REST API reference](_09rest-api.md): REST API reference.
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
9. [REST API reference](_09-rest-api.md): REST API reference.
```

File: source/includes/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+You can also visit the [Frequently Asked Questions (FAQ)](faq.md), where you'll find questions and common issues people
```
Comment (tainguyenbui) [inline source/includes/README.md]:
```suggestion
You can also visit the [Frequently Asked Questions (FAQ)](_faq.md), where you'll find questions and common issues people
```

File: source/includes/README.md (hunk 6 above)
Comment (adrian-lorenzo) [reply source/includes/README.md]: This file is outdated and not necessary. I will remove it!

Sorry @tainguyenbui!

**Pull request merged**